packages/ui5-proxy-middleware/package.json

```diff
@@ -35,12 +35,11 @@
         "yaml": "2.0.0-10"
     },
     "devDependencies": {
+        "@sap-ux/ui5-config": "workspace:*",
         "@types/express": "4.17.13",
-        "@ui5/server": "2.4.0",
-        "@ui5/project": "2.6.0",
```

Comment:
Since we are removing the dependencies to `@ui5`, we can also remove this one here https://github.com/SAP/open-ux-tools/blob/main/package.json#L51 😃

Reply:
Done